gulpfile.js

```diff
@@ -4,6 +4,7 @@ const gulp = require('gulp');
 const util = require('gulp-util');
 const ngrok = require('ngrok');
 const nodemon = require('gulp-nodemon');
+const patchRule = require('./lib/patchRule').default;
```

Comment:
Why opt to create this rather than leverage https://github.com/auth0-extensions/auth0-account-link-extension/blob/master/modifyRule.js?

Reply:
My fault, didn't noticed that one 😕

Reply:
No worries! I was just concerned I made some glaring mistakes in mine :D